hazelcast/scripts/cleanDownloaded.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 HAZELCAST_BRANCH="$1"
+HAZELCAST_BRANCH="distributed_scheduled_exec"
```

Comment:
You already changed pom.xml properties:

        <hazelcast.git.repo>tkountis</hazelcast.git.repo>      <hazelcast.git.branch>feature/distributed_scheduled_exec</hazelcast.git.branch>

Why do you still need this change here?